### PR TITLE
Add dlrm-preprocessing readme

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: pyzoo/zoo/friesian/feature/table.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def gen_string_idx(self, columns, freq_limit):
 
         :param columns: str or a list of str, target columns to generate StringIndex.
         :param freq_limit: int, dict or None. Categories with a count/frequency below freq_limit
-               will be ommited from the encoding. Can be represented as both an integer,
+               will be omitted from the encoding. Can be represented as both an integer,
                dict or None. For instance, 15, {'col_4': 10, 'col_5': 2} etc.
```
Comment (hkvision) [inline pyzoo/zoo/friesian/feature/table.py:217]: explain what does None mean.

File: pyzoo/zoo/examples/friesian/feature/dlrm_preprocessing.py [hunk 2]
```diff
@@ -68,7 +68,10 @@ def _parse_args():
         choices=['overwrite', 'errorifexists'],
         default='errorifexists')
 
-    parser.add_argument('--frequency_limit', type=str)
+    parser.add_argument('--frequency_limit', type=str,
+                        help="Categories with a count/frequency below frequency_limit will be "
+                             "omitted from the encoding. For instance, '15', '_c14:15,_c15:16', "
+                             "etc")
```
Comment (hkvision) [inline pyzoo/zoo/examples/friesian/feature/dlrm_preprocessing.py:74]: Give a default value 15?

File: pyzoo/zoo/examples/friesian/feature/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+## Prepare the data
+You can download Criteo data from <https://ailab.criteo.com/download-criteo-1tb-click-logs-dataset/>.
+ 
+After you download the files, you should convert them to parquet files with the name day_x.parquet(x
+=0-23), and put all parquet files in one folder.
+
+
+## Preprocess
+* Spark local, example command
```
Comment (hkvision) [inline pyzoo/zoo/examples/friesian/feature/README.md:21]: Add some remark something like, for local mode, we can use the first several days to have a trial?

**Review by hkvision — APPROVED** [review]

**Comment by cyita** [comment]
Jenkins passed http://10.239.47.210:18888/job/ZOO-PR-Validation/5474/

**Pull request merged**